LanternExtractor/EQ/Archive/ArchiveType.cs

```diff
@@ -0,0 +1,9 @@
+namespace LanternExtractor.EQ.Archive
+{
+    public enum ArchiveType
+    {
+        Unknown,
```

Comment:
How often do we actually run into an `Unknown` type archive? Also, can you assign numerical values to these.

Reply:
Realistically I think unknown will really only get hit when non archive files get renamed to one of the archive extensions or when the files are damaged to the point where the the first 4 (or second 4) bytes are incorrect.